j2/vars/custom-webiny-rules.yaml

```diff
@@ -0,0 +1,539 @@
+            excludeFromSearch: true
+          Type_Of_Natural_Person:
+            val: "${mapeamentoDeProcessosCategoriaDosTitularesDosDados}"
+            excludeFromSearch: true
+#          Data_Provider:
+#            val: "${}"
+#            excludeFromSearch: true
+#          Why_Is_It_Collected:
+#            val: "${mapeamentoDeProcessosFinalidadeDoProcesso}"
+#            excludeFromSearch: true
+#          Secondary_Purpose_Of_Collection:
+#            val: "${}"
+#            excludeFromSearch: true
+#          Country_Where_Stored:
+#            val: "${}"
+#            excludeFromSearch: true
+#          Participants:
+#            vals: "${}"              TODO: what to do with that ?!
+#            excludeFromSearch: true
+#          Info_Shared_With_Third_Parties:
+#            vals: "${}"
+#            excludeFromSearch: true
+#          Info_Shared_With_Third_Parties_International:
+#            vals: "${}"
+#            excludeFromSearch: true
+#          Info_Share_Justification:
+#            val: "${}"              TODO: não temos campo para essa questão
+#            excludeFromSearch: true
+#          Info_Share_Justification_International:
+#            val: "${}"
+#            excludeFromSearch: true
+#          Number_Of_Natural_Person:
+#            val: "${}"
+#            excludeFromSearch: true
+          Products_And_Services:
+            val: "${mapeamentoDeProcessosDadosPessoaisEstaoRelacionadosProdutosServicos}"
+            excludeFromSearch: true
+#          Rights:
+#            val: "${}"               #TODO: Check with @Leo if it is needed into Webiny

```

Comment:
TODO: loop over sensitive data , similar to lawful basis, if not, then ... conditions!

Reply:
don't think this can be done!